platforms/windows/src/tests/subclassed.rs

```diff
@@ -61,16 +60,26 @@ impl ActionHandler for NullActionHandler {
 // This module uses winit for the purpose of testing with a real third-party
 // window implementation that we don't control.
 
+#[cfg(target_os = "windows")]
```

Comment:
not sure that's the best way to handle that. I assumed it makes little sense to test other platforms in platform/windows 🤔

Reply:
The `accesskit_windows` crate compiles just fine on Linux, and probably on macOS as well. Therefore one might try to run these tests outside of a Windows environment. So yes, this check makes sense to me.